twitter/twitter.php

```diff
@@ -48,12 +48,13 @@
  *     we do not need "Twitter as login". When you've registered the app you get the
```

Comment:
Could you please move the configuration information into a `README.md` file so that it is displayed in the git hosting and admin panel.